UnityProject/Assets/Scripts/Objects/NukeInteract.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections;
+using PlayGroup;
+using PlayGroups.Input;
+using UI;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.Video;
+
+public class NukeInteract : InputTrigger
+{
+    public float cooldownTimer = 2f;
+    public string interactionMessage;
+    public string deniedMessage;
+    [SyncVar]
+    private int nukeCode;
+    private GameObject Player;
+    void Start()
+    {
+        //calling the code generator and setting up a 10 second timer to post the code in syndicate chat
+        codeGenerator();
+        StartCoroutine(WaitForShit());
+    }
```

Comment:
You want this to be:
``
public override void OnStartServer(){
base.OnStartServer();
}
``
So that this start method only runs on the server